packages/kolibri-tools/lib/cli.js

```diff
@@ -226,8 +227,15 @@ program
     list,
     []
   )
+  .option('--kds', 'Flag to use local kds', false)
```

Comment:
I think that maybe the description here isn't entirely true. You don't _need_ this flag to run with local KDS but rather you just need this flag if you want to get errors related to your attempts to use local KDS.

It might be better to call this something like `--require-kds-path` to better indicate what it is really doing.